hollow/src/test/java/com/netflix/hollow/api/codegen/AbstractHollowAPIGeneratorTest.java

```diff
@@ -43,6 +49,24 @@ void runGenerator(String apiClassName, String packageName, Class<?> clazz,
         HollowCodeGenerationCompileUtil.compileSrcFiles(sourceFolder, clazzFolder);
     }
 
+    void assertNonEmptyFileExists(String relativePath) throws IOException {
+        if (relativePath.startsWith("/")) {
+            throw new IllegalArgumentException("Relative paths should not start with /");
+        }
+        try (BufferedReader br = new BufferedReader(
```

Comment:
Not sure of if wanna check if the file exists and it's length > 0 or there is at least one line:
```
File f = new File(sourceFolder + "/" + relativePath);
assertTrue(f.exists() && f.length() > 0L);
```
or
```
try (Stream<String> lines = FIles.lines(Paths.get("sourceFolder + "/" + relativePath")) {
  assertTrue(lines.findFirst().isPresent());
}
```

Reply:
I was going for length > 0 - thanks, will update